drv/hisi_comp.c

```diff
@@ -883,8 +886,6 @@ static int hisi_zip_comp_send(handle_t ctx, struct wd_comp_msg *msg, void *priv)
 		return ret;
 	}
 
-	hisi_qm_enable_interrupt(ctx, msg->is_polled);
-
```

Comment:
You didn't explain why to remove hisi_qm_enable_interrupt().

Reply:
add needed information to commit message now.